content/articles/how-to-use-a-local-dns-resolver-to-redirect-all-dns-requests/index.md

```diff
@@ -0,0 +1,115 @@
+
+You may want to go to the firewall rules page for the interface where you've put a NAT port forward rule after clicking `Save`. Firewall > Rules > IoT, for example. A rule was automatically inserted at the end of the list, as you can see. Depending on the other rules you have defined, this auto-generated rule may need to be moved to the top of the list. The other rule(s) should not interfere with the correct functioning of the rule.
+
+However, if you have many networks, this rule must be repeated for each network where DNS requests are redirected. Continued repetition could become dull and irritating. Also, if you decide to change the law in the future, it will require extra upkeep. Fortunately, there's a technique to create a single rule for numerous interfaces that's simple.
+
+### Configuration of multiple Interfaces
+You can construct a firewall group to apply a single rule to numerous interfaces. Using firewall groups, you can arrange several interfaces into a single group and write rules that apply to all of the interfaces in the group at once. When it comes to the ruling hierarchy, group rules take precedence over a set of interface rules. DNS redirection rules are likely to be executed before interface rules. Therefore you shouldn't have any problems. It is important to keep in mind the rule processing order while developing group rules for multiple interfaces.
+
+Go to the `Firewall > Groups` page to create a rule group. Then select "Add" from the drop-down menu.
+
+![Add](/engineering-education/how-to-use-a-local-dns-resolver-to-redirect-all-dns-requests/image-03)
+
+Assign the firewall group a `Name` before selecting the interfaces/networks to which you want the DNS requests redirected. `commands` is a descriptive name for the group in my example. For some interfaces, you may not require or want redirection; therefore, you can uncheck the box. You may, for example, have a network that is completely disconnected from the Internet. There may be networks where you would rather prohibit external DNS requests than reroute them. For the group to be created, click on `Save`.
+
+![Save](/engineering-education/how-to-use-a-local-dns-resolver-to-redirect-all-dns-requests/image-04)
+
+Just as before, you can create a redirection by selecting the firewall group instead of an interface. Click the `Add` button on the "Firewall > NAT > Port Forward" page. Please enter the following values:
+
+![Value entry](/engineering-education/how-to-use-a-local-dns-resolver-to-redirect-all-dns-requests/image-05)
+
+The redirection from the NAT port forward should have been immediately added to the rule list on the `Firewall > Rules > CommonDNS` page.
+
+![Page view](/engineering-education/how-to-use-a-local-dns-resolver-to-redirect-all-dns-requests/image-06)

```

Comment:
It seems that image 5 is the one that's supposed to be here

Reply:
Changed the image